### PR TITLE
moonriver v0.26.0

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

update macos snippet to v0.26.0 for moonriver & moonbase

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]

**Comment by eshaben** [comment]
waiting for approval on https://github.com/PureStake/moonbeam-docs/pull/401 to merge

**Pull request merged**
